src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionsCache.cs

```diff
@@ -36,15 +36,15 @@ internal class CodeActionsCache
         /// <summary>
         /// Current list of cached items.
         /// </summary>
-        private readonly List<CodeActionsCacheItem> _cachedItems = new List<CodeActionsCacheItem>();
+        private readonly List<CodeActionsCacheItem> _cachedItems = new();
 
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
         public CodeActionsCache()
         {
         }
 
-        public async Task UpdateActionSetsAsync(
+        internal async Task UpdateActionSetsAsync(
```

Comment:
curious hwy?

Reply:
I was just looking through the CodeActionsCache class as reference and noticed that these didn't need to be public (and I'm not sure why I made them public in the first place). There should be no behavioral change here though.